File: Revision_classes/09_1_SM&NSM/Test1.java
class A
{
	static int a = 10;
	static{
		System.out.println("class A SB is executed");
	  }
		public static void main(String[] args) 
	{
		System.out.println("A main");
	}
}

class  Test1 extends A
{
	static int b = 20;
		static{
		System.out.println("class Test SB is executed");
	   }
	public static void main(String[] args) 
	{
		System.out.println("Test main");
		System.out.println("a: " + a);
		System.out.println("b:" + b);
	}
}

File: React/app8/src/Test1.js
export default function Test1() {
    const callToApi = () => {
        fetch('http://localhost:9090/test/t1')
           // .then(() => console.log('ok'))
            .then((response) => response.text())
              .then((response) => console.log(response))
            .catch(() => console.log('error'))
    }
    return (<>
        <button onClick={callToApi} type='button'>Call to API 123</button>
    </>)
}